### ISSUE TITLE
[Warning message] UserWarning: None of the inputs have requires_grad=True. Gradients will be None

### ISSUE THREAD

**Issue by y-rok** [Victorwz/LLaVA-Llama-3#4]
```
/usr/local/lib/python3.10/dist-packages/torch/utils/checkpoint.py:429: UserWarning: torch.utils.checkpoint: please pass in use_reentrant=True or use_reentrant=False explicitly. The default value of use_reentrant will be updated to be False in the future. To maintain current behavior, pass use_reentrant=True. It is recommended that you use use_reentrant=False. Refer to docs for more details on the differences between the two variants.
  warnings.warn(
/usr/local/lib/python3.10/dist-packages/torch/utils/checkpoint.py:61: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn(

```

Have you got this warning message? 
Is there any problem if i just ignore this warning message?

**Comment by Victorwz** [comment]
Hi, it appears only in iteration 0 and has no effects to the training.
